generate_data.py

```diff
@@ -34,6 +35,9 @@ def generate(scale: float, path: os.PathLike) -> None:
                 SET s3_session_token='{creds.token}';
                 """
             )
+        else:
+            path = pathlib.Path(path)
+            path.mkdir(parents=True, exist_ok=True)
```

Comment:
Hmm I'm curious why this is needed given the `mkdir` calls later in this function, but I'll probably find out while running this code locally. Feel free to fill me in if you recall why this was added

Reply:
This might just be a merge issue.